### PR TITLE
py-fastavro: update to 0.14.3

### PR DESCRIPTION
**Pull request by esafak**
###### Description
Updated and added py36 support

###### Tested on
macOS 10.x

###### Verification 
- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?

### CODE REVIEW

**Comment by esafak** [comment]
Re-added MD5 hash and tested an install.

File: python/py-fastavro/Portfile [hunk 1]
```diff
@@ -22,14 +22,14 @@ homepage            https://fastavro.readthedocs.org/
 distname            ${_name}-${version}
 master_sites        pypi:${_n}/${_name}/
 
-checksums           md5     af74d03d1fd3a2384ca655e08650ce76 \
-                    rmd160  546f374c2e10a4f189c925dacca5525c89104f62 \
-                    sha256  d6404f006e5b734021baed993c957ba12e98ef330a07ad9075c221738567f0cb
+checksums           md5 	6bab0b9ec952784c81d7efe56c8ea9da \
+					rmd160  1b067231076c45a0aecc97c3d28a7ffbd8959b49 \
+                    sha256  db470e239796a7c873d42dbba70f56af6d97a335b2e5b1249670fe849cdff4ef
```
Comment (raimue) [inline python/py-fastavro/Portfile]: Please use spaces, not tabs.

File: python/py-fastavro/Portfile [hunk 2]
```diff
@@ -22,14 +22,14 @@ homepage            https://fastavro.readthedocs.org/
...
 
-python.versions     27 34 35
+python.versions     27 34 35 36
 
 if {${name} ne ${subport}} {
-    depends_build-append    port:py${python.version}-setuptools
+    depends_build-append	port:py${python.version}-setuptools
```
Comment (raimue) [inline python/py-fastavro/Portfile]: Unnecessary whitespace change.

**Comment by esafak** [comment]
Done, @raimue and updated to 0.14.3

**Comment by raimue** [comment]
* Converted tabs to spaces
* Fixed typo in categories in 1b522cb4e9
